docker/cluster-test/buildspec.yaml

```diff
@@ -13,7 +13,7 @@ phases:
     commands:
       - echo Build started on `date`
       - echo Building the Docker image...
-      - docker/cluster-test/build.sh
+      - ephemeral_container=1 docker/cluster-test/build.sh
```

Comment:
Can you give some more details what is different about code build env?

Reply:
This command fails to run in the codebuild env `docker run --name libra_cluster_test_builder_container -d -i -t -v $DIR/../..:/libra libra_cluster_test_builder bash`

It gives the error : `the input device is not a TTY`

Reply:
I see.
It's perfectly fine to just remove `-t` flag for both cases then - this way you won't need ephemeral_container var and I don't think it will hurt anything